docs/examples/fork-process-model/flask-uwsgi/requirements.txt

```diff
@@ -2,7 +2,7 @@ click==8.1.7
 Flask==2.3.3
 googleapis-common-protos==1.52.0
 grpcio==1.56.2
-gunicorn==22.0.0
+gunicorn==23.0.0
```

Comment:
```suggestion

```

Reply:
I think we can just drop gunicorn from there since it's not used in this example